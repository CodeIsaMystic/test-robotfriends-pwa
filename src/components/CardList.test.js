//import { shallow } from 'enzyme';

import React from 'react';
import { create } from "react-test-renderer";
import CardList from './CardList';




const mockRobots = [
  {
    id: 1,
    name: 'Jon Snow',
    username: 'JonJon',
    email: 'jon@gmail.com'
  }
];

const cardList = create(<CardList robots={mockRobots} />);


it('to render CardList Component with react test renderer', () => {

  expect(cardList.toJSON()).toMatchSnapshot();
});
