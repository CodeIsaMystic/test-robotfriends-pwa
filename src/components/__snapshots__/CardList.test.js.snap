// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`to render CardList Component with react test renderer 1`] = `
<div>
  <div
    className="tc grow bg-light-green br3 pa3 ma2 dib bw2 shadow-5"
  >
    <img
      alt="robots"
      src="https://robohash.org/1?size=200x200"
    />
    <div>
      <h2>
        Jon Snow
      </h2>
      <p>
        jon@gmail.com
      </p>
    </div>
  </div>
</div>
`;
